templates/report.html.jinja

```diff
@@ -326,7 +341,7 @@
                     <thead class="border-t">
                         <tr>
                             <th class="pt-12" colspan="2">
-                                Validation errors observed
+                                MobilityData GTFS Schedule Validator notices observed
```

Comment:
We could add a link to the validator here, for example https://gtfs-validator.mobilitydata.org/?

Reply:
Done! I opted to link to https://gtfs.org/schedule/validate/, which has a bit more summary info, and then links to the actual validator tool